gitgud/levels/reworking/_interactiverebase/instructions.txt

```diff
@@ -0,0 +1,13 @@
+This practice level will familiarize with using interactive rebasing.
+>>>
+This is an alternative way of using the "rebase" command with "-i" right after it. For example, "git rebase -i HEAD~4" allows you to change the last 4 commits with the text editor.
+>>>
+When you include this option, git will open a text editor to show which commits will be copied below the rebase target.
+In the text editor, you're given the ability to do three things: reorder commits, omit commits, and squash commits(combine commits).
```

Comment:
There's actually 10 things that you can do. Do git rebase -i HEAD, and you'll be able to see all the options before doing anything. 

You can tell them to try and do that too in order to see the different options too, then go ahead and tell them to actually complete the challenge with `git rebase -i whatever`.

You can tell them that the most common things are reording commits, dropping that word, or at lease clarify that git uses that word, and squashing.